browserstack.json

```diff
@@ -0,0 +1,41 @@
+{
+    "auth": {
+        "username": "tilmanvatteroth1",
+        "access_key": "Sp3viAhcgqcRMuZC1Mn1"
+    },
+    "browsers": [
+        {
+            "browser": "chrome",
+            "os": "Windows 10",
```

Comment:
Do we also want to add Mac tests here?
(Linux is sadly not supported)

Reply:
Cypress doesn't support safari (yet)

Reply:
Hm, too bad. I only had a quick view into the UI and saw "Windows" and "Mac". So I thought, they'd support Safari.

Reply:
BrowserStack supports safari (afaik). But cypress doesn't.